cmd/ctr/run.go

```diff
@@ -94,10 +93,14 @@ var runCommand = cli.Command{
 
 			ctx, cancel = appContext(context)
 			id          = context.Args().Get(1)
+			imageRef    = context.Args().First()
 			tty         = context.Bool("tty")
 		)
 		defer cancel()
 
+		if imageRef == "" {
```

Comment:
it isn't here; `newContainer()` in *cmd/ctr/run_unix.go* uses the ref without any checking for empty string.